<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReadMNIST</class>
 <widget class="QWidget" name="ReadMNIST">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>187</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReadMNIST</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="spinBox">
         <property name="maximum">
          <number>59999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="statusLbl">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid rgba(5, 255, 5, 0.7);
border-radius: 5px;
background-color: rgba(113, 222, 78, 100);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
       <property name="indent">
        <number>11</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DigitViewer" name="digitViewer" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DigitViewer</class>
   <extends>QWidget</extends>
   <header>digitviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ReadMNIST</receiver>
   <slot>readData(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>readData()</slot>
  <slot>nextImage()</slot>
  <slot>prevImage()</slot>
  <slot>readData(int)</slot>
 </slots>
</ui>
